src/test/resources/org/rust/ide/formatter/fixtures/spacing_after.rs

```diff
@@ -207,8 +207,7 @@ impl<T: PartialEq> Vec<T> {
     }
 }
 
-pub fn from_elem<T: Clone>(elem: T, n: usize) -> Vec<T> {
-}
+pub fn from_elem<T: Clone>(elem: T, n: usize) -> Vec<T> {}
```

Comment:
Also I'm not 100% sure that this change is also desired. I have noticed that rustfmt always transforms `{<whitespace>}` to `{}`, so I have implemented similar behaviour here (previously it allowed 1 `\n` between braces).